### DISCUSSION TITLE
Ubuntu 22.04 DuckDb CLI using Azure Extensions - Problem with the SSL CA cert (path? access rights?) - 0.9.1 and 0.9.2 both version

### DISCUSSION THREAD

**Discussion by radix-naimsulejmani** [duckdb/duckdb#9675] [Q&A]
I am running DuckDB CLI on Ubuntu 22.04 and I receive this error when I try to query Azure blob storage using Azure extensions

`
INSTALL azure;
LOAD azure;
SET  azure_storage_connection_string = 'CONNECTION_SRING OF AZURE BLOB STORAGE';

SELECT * FROM 'azure://container_name/filename.parquet';

`

It shows an error:
Error: Invalid Error: Fail to get a new connection for: https://[domain].blob.core.windows.net. Problem with the SSL CA cert (path? access rights?)

**Comment by jaihind213** [comment]
quick fix mentioned in https://github.com/duckdb/duckdb_azure/issues/7

`
mkdir -p /etc/pki/tls/certs
ln -s /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
`

**Comment by rubinboban** [comment]
SET azure_transport_option_type = 'curl'; worked for me
